src/pipeline/activator/aviris-l2/main.py

```diff
@@ -45,24 +45,27 @@ def main():
     parser.add_argument(
         "aviris_stac_id",
         type=str,
-        help="STAC Item ID to process from the {} STAC collection".format(
-            AVIRIS_ARCHIVE_COLLECTION_ID
-        ),
+        help="STAC Item ID to process from the STAC collection"
+    )
+    parser.add_argument(
+        "--aviris_collection_id",
+        type=str,
+        default=AVIRIS_ARCHIVE_COLLECTION_ID,
     )
     parser.add_argument(
         "--franklin-url",
         type=str,
         default=os.environ.get(
-            "AAL2_FRANKLIN_URL", "http://franklin:9090"
```

Comment:
We'll revert it back once it would be obvious that we really need unique ENV var names.